tests/test_opentime.py

```diff
@@ -524,95 +538,96 @@ def test_frames_with_int_fps(self):
         for fps in (24, 30, 48, 60):
             t1 = otio.opentime.from_frames(101, fps)
             t2 = otio.opentime.RationalTime(101, fps)
-            self.assertEqual(t1, t2)
+            assert (t1,t2)
 
     def test_frames_with_nonint_fps(self):
         for fps in (23.98, 29.97, 59.94):
             t1 = otio.opentime.from_frames(101, fps)
             t2 = otio.opentime.RationalTime(101, fps)
-            self.assertEqual(t1, t2)
+            assert (t1,t2)
 
     def test_seconds(self):
         s1 = 1834
         t1 = otio.opentime.from_seconds(s1)
-        self.assertEqual(t1.value, 1834)
-        self.assertEqual(t1.rate, 1)
+        assert (t1.value == 1834)
+        assert (t1.rate == 1)
         t1_as_seconds = otio.opentime.to_seconds(t1)
-        self.assertEqual(t1_as_seconds, s1)
-        self.assertAlmostEqual(float(t1.value) / t1.rate, s1)
+        assert (t1_as_seconds == s1)
+        assert (float(t1.value) / t1.rate, s1)
```

Comment:
These assertions don't quite match, the unittest version was using `assertAlmostEqual`, the updated version is asserting a tuple with two elements, which will always pass.

This should probably use [`pytest.approx`](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-approx):
```python
assert (float(t1.value) / t1.rate) == pytest.approx(s1)
```